gocron.go

```diff
@@ -48,6 +48,7 @@ const (
 
 var (
 	locker Locker
+	th     = newTimeHelper()
```

Comment:
I believe this would be better injected in the scheduler instead of being a global instance

Reply:
Sure. Makes sense. Will update it in some time

Reply:
`job` also uses the time functions. So, injecting it there as well.